guide/designing-products/open-design.md

```diff
@@ -51,68 +52,105 @@ Illustration sources
-* create a design resource library like an icon set or a UI kit
-* help test an open-source product and provide feedback
-* [contribute](/guide/contribute/) to the Bitcoin Design Guide
+Open design communities can have multidisciplinary teams with designers, developers, copywriters, translators, project managers.... These diverse groups can develop stronger solutions through varied points of view.
 
-Whatever you choose to do, open design projects are voluntary public contributions. From this work, a network of people and projects emerge connected by personal relationships, specialized communities, and sharing of work. This is different from corporate structures and requires individuals to think and act more independently.
+## Helpful skills
 
-In open-source software, public review and debate lead to better products and services. Open-source development has existed for decades, while open design is not widely known or practiced. Designing for bitcoin products, especially for newcomers in an open design setting, can be disorienting and require adjustments to how they work and collaborate. No longer is work the exclusive product of them or a small, localized team; instead, it becomes part of a public conversation.
+- Ethics
+- Personal responsibility
+- Independence & self-will
+- Communication
+- Open-mindedness
+- Respectful feedback
+- Time management
+- Teamwork
+- Openness & transparency
 
-Designers, developers, translators, project managers, and others work together in the open. Multidisciplinary collaboration, openness, and diversity of people involved make sure that the best solution wins no matter where it comes from.
+## Bitcoin has a spirit of openness
 
-##  Open design and Bitcoin
+We could argue Bitcoin needs open design to stay true to its mission.
 
-The spirit of openness is deeply embedded into Bitcoin; the source code is public, participation in the network is public and unrestricted, and transferring value via the Bitcoin protocol is permissionless. Thousands of projects have borrowed ideas, techniques, and code from Bitcoin to create their open-source experiments. For designers who aim to work collaboratively, it’s a robust foundation.
+* Bitcoin is public and open source
+* It has public & unrestricted network participation
+* It allows permissionless transfer of value in the Bitcoin protocol
+
+Thousands of projects have borrowed ideas, techniques, and code from Bitcoin. Countless open-source experiments have arisen. To a designer, Bitcoin is a robust foundation for collaboration. A foundation that comes with a responsibility toward the benefit of society.
+
+This ethical responsibility encompasses application design. Most people interact with applications instead of the underlying Bitcoin technical protocol. If someone controls access to an application, it won't matter if the protocol is open. It is up to the global community to build principled Bitcoin applications.
```

Comment:
```suggestion
This ethical responsibility encompasses application design. Most people interact with applications instead of the underlying Bitcoin technical protocol. If someone controls access to an application, it won't matter if the protocol is open. It is up to the global community to build [principled Bitcoin applications](guide/designing-products/principles/)
```